src/vs/workbench/contrib/scm/browser/scm.contribution.ts

```diff
@@ -230,6 +231,34 @@ KeybindingsRegistry.registerCommandAndKeybindingRule({
 	}
 });
 
+KeybindingsRegistry.registerCommandAndKeybindingRule({
+	id: 'scm.viewPriorCommit',
+	description: { description: localize('scm view prior commit', "SCM: View Prior Commit"), args: [] },
+	weight: KeybindingWeight.WorkbenchContrib,
+	when: ContextKeyExpr.has('scmRepository'),
+	primary: KeyMod.Alt | KeyCode.UpArrow,
```

Comment:
Should this work like the search viewlet instead? Pressing up while on the first line should go back in history, etc. Do we need an `Alt` modifier?

Reply:
I was planning to replace that with [this ](https://github.com/microsoft/vscode/issues/107458#issue-709102124)logic